lib/apiservers/engine/backends/container.go

```diff
@@ -1097,31 +1099,36 @@ func (c *Container) defaultScope() string {
 	return defaultScope.scope
 }
 
-func (c *Container) findPortBoundNetworkEndpoint(hostconfig *containertypes.HostConfig, endpoints []*models.EndpointConfig) *models.EndpointConfig {
+func (c *Container) findPortBoundNetworkEndpoint(hostconfig *containertypes.HostConfig, endpoints []*models.EndpointConfig) (*models.ScopeConfig, *models.EndpointConfig) {
 	if len(hostconfig.PortBindings) == 0 {
-		return nil
+		return nil, nil
 	}
 
 	// check if the port binding network is a bridge type
 	listRes, err := PortLayerClient().Scopes.List(scopes.NewListParamsWithContext(ctx).WithIDName(hostconfig.NetworkMode.NetworkName()))
 	if err != nil {
 		log.Error(err)
-		return nil
+		return nil, nil
 	}
 
-	if len(listRes.Payload) != 1 || listRes.Payload[0].ScopeType != "bridge" {
+	if l := len(listRes.Payload); l != 1 {
+		log.Warnf("found %d scopes", l)
+		return nil, nil
+	}
+
+	if listRes.Payload[0].ScopeType != "bridge" {
```

Comment:
Should use the constant defined in the `github.com/vmware/vic/lib/portlayer/constants` package.